src/internal/utils/bucketHelper.ts

```diff
@@ -799,3 +805,24 @@ export function getAllMeasuresShowOnSecondaryAxis(buckets: IBucket[]): IBucketIt
 export function getItemsLocalIdentifiers(items: IBucketItem[]): string[] {
     return items.map((item: IBucketItem) => get(item, "localIdentifier", ""));
 }
+
+const getAvailableMeasureBucketsLocalIdentifiers = (type: VisType) =>
+    (type === VisualizationTypes.BULLET && getBulletChartSupportedMeasureBuckets()) || [];
```

Comment:
same here

Reply:
We agreed to leave it as it is, am I right?